lab_1_keywords_tfidf/main.py

```diff
@@ -82,7 +117,17 @@ def calculate_tf(frequencies: dict[str, int]) -> Optional[dict[str, float]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(frequencies, dict):
+        return None
+    total_words = 0
+    tf_dict = {}
+    for token, frequency in frequencies.items():
+        if not isinstance(token, str):
+            return None
+        total_words += frequency
```

Comment:
try to use `sum` and do this step in one line

Reply:
Done